rust/cardano-chain-follower/Cargo.toml

```
@@ -15,13 +15,12 @@ pallas = { version = "0.30.1", git = "https://github.com/input-output-hk/catalys
 pallas-hardano = { version = "0.30.1", git = "https://github.com/input-output-hk/catalyst-pallas.git", rev = "9b5183c8b90b90fe2cc319d986e933e9518957b3" }
 pallas-crypto = { version = "0.30.1", git = "https://github.com/input-output-hk/catalyst-pallas.git", rev = "9b5183c8b90b90fe2cc319d986e933e9518957b3" }
 
-# mithril-client = { version = "0.10.4", git = "https://github.com/input-output-hk/mithril", rev = "c6c7ebafae0158b2c1672eb96f6ef832fd542f93", default-features = false, features = [
 mithril-client = { version = "0.10.4", default-features = false, features = [
     "full",
     "num-integer-backend",
 ] }
-
-rbac-registration = { version = "0.0.2", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "v0.0.8" }
+cardano-blockchain-types = { version = "0.0.1", git = "https://github.com/input-output-hk/catalyst-libs.git", branch = "fix/cip36" }
```

Need to fix the import from this branch
```suggestion
cardano-blockchain-types = { version = "0.0.1", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250108-00" }
```